static/app/views/starfish/views/appStartup/index.tsx

```diff
@@ -0,0 +1,186 @@
+import styled from '@emotion/styled';
+
+import Alert from 'sentry/components/alert';
+import LoadingContainer from 'sentry/components/loading/loadingContainer';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import SearchBar from 'sentry/components/performance/searchBar';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {NewQuery} from 'sentry/types';
+import {defined} from 'sentry/utils';
+import EventView from 'sentry/utils/discover/eventView';
+import {DiscoverDatasets} from 'sentry/utils/discover/types';
+import {decodeScalar} from 'sentry/utils/queryString';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import useRouter from 'sentry/utils/useRouter';
+import {prepareQueryForLandingPage} from 'sentry/views/performance/data';
+import {getTransactionSearchQuery} from 'sentry/views/performance/utils';
+import {useReleaseSelection} from 'sentry/views/starfish/queries/useReleases';
+import {SpanMetricsField} from 'sentry/views/starfish/types';
+import {formatVersionAndCenterTruncate} from 'sentry/views/starfish/utils/centerTruncate';
+import {appendReleaseFilters} from 'sentry/views/starfish/utils/releaseComparison';
+import {getFreeTextFromQuery} from 'sentry/views/starfish/views/screens';
+import {
+  ScreensTable,
+  useTableQuery,
+} from 'sentry/views/starfish/views/screens/screensTable';
+
+const MAX_TABLE_RELEASE_CHARS = 15;
+
+type Props = {
+  additionalFilters?: string[];
+  chartHeight?: number;
+};
+
+function AppStartup({additionalFilters}: Props) {
+  const pageFilter = usePageFilters();
+  const {selection} = pageFilter;
+  const location = useLocation();
+  const organization = useOrganization();
+  const {query: locationQuery} = location;
+
+  const {
+    primaryRelease,
+    secondaryRelease,
+    isLoading: isReleasesLoading,
+  } = useReleaseSelection();
+  const truncatedPrimary = formatVersionAndCenterTruncate(
+    primaryRelease ?? '',
+    MAX_TABLE_RELEASE_CHARS
+  );
+  const truncatedSecondary = formatVersionAndCenterTruncate(
+    secondaryRelease ?? '',
+    MAX_TABLE_RELEASE_CHARS
+  );
+
+  const router = useRouter();
+
+  const query = new MutableSearch([
+    'event.type:transaction',
+    'transaction.op:[app.start.cold,app.start.warm]',
```

Comment:
Should we just leave this as `transaction.op:ui.load` for now so that table/visualizations are populated?

Reply:
Good call, since we're reconsidering renaming the root transaction, I can swap this back for now